### ISSUE TITLE
Feature: Parallel execution of Publisher.after-hooks

### ISSUE THREAD

**Issue by rrrene** [rrrene/elixirstatus-web#12]
When thinking about it, we should execute the stuff `Publisher.after_create/1` and `Publisher.after_update/1` do in a parallel fashion, as this does not have to finish before the Phoenix server can send a response.

But it has to ensure execution, so a simple `spawn fn -> ...` won't do the trick.
